assets/css/localstyles.css

```diff
@@ -0,0 +1,52 @@
+/* PirataOne/pirata-one-regular - latin */
```

Comment:
I think this file can be dropped entirely in favor of loading each font indepedently.

This isn't the *ideal* method since it adds additional unnecessary HTTP connections but the way the forthcoming webfonts api works means this will problem that will go away once we switch APIs (pending core changes of course).